site/gatsby-site/server/remote.ts

```diff
@@ -47,7 +47,7 @@ const ignoreTypes = [
     'CreateVariantInput',
 
     // 'Incident',
-    'IncidentQueryInput',
+    // 'IncidentQueryInput',
```

Comment:
Does the `TODO` on line 27 and these comments make sense? I'm not clear

Reply:
Yes, what's happening there is that the subscription -> user relation breaks if we ignore the `UserQueryInput` type.